specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/0.1-preview/confidentialledger.json

```diff
@@ -349,6 +349,36 @@
         }
       }
     },
+    "/app/subLedgerIds": {
+    "get": {
+        "operationId": "ConfidentialLedger_GetSubLedgerIds",
+        "x-ms-examples": {
+            "GetSubLedgerIds": {
+                "$ref": "./examples/ledger/GetSubLedgerIds.json"
+            }
+        },
+        "summary": "Retrieves the ids of subledgers",
```

Comment:
nit: summary can be a bit more descriptive e.g. "Retrieves a list of sub-ledgers present in the Confidential Ledger"